src/pages/service.tsx

```diff
@@ -105,6 +105,14 @@ const ServicePage = ({
     )
   )}`;
 
+  if ((apiUrlParams.toLowerCase().includes("localhost")) || (apiUrlParams.toLowerCase().includes("127.0.0.1"))) {
+        alert("Wrong URL! Please check the URL and enter it correctly.");
+        console.error("Wrong URL");
+	apiUrlParams = apiUrlParams.replace("localhost", "");
+	apiUrlParams = apiUrlParams.replace("127.0.0.1", "");
+  }
```

Comment:
Please do not duplicate validation. Sanitation should happen only in one place to avoid inconsistent enforcement.